### PR TITLE
LIVE-4430 feature ios disable nft FF

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Disable NFT in iOS to be apple compliant.
Also it's require to add variable on firebase : `feature_disable_nft_ledger_market`
`feature_disable_nft_rarible_opensea`
`feature_disable_nft_send`

with value : 
```
{"enabled": true}
```

### ❓ Context

- **Impacted projects**: `live-common, live-mobile` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-4430 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

Screenshot 2022-11-04 at 10 16 27
Screenshot 2022-11-04 at 10 16 53

https://user-images.githubusercontent.com/33897120/199937052-14e550bb-0052-40d1-a7d9-b5d7bc7998ca.mov

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx
Comment (lvndry) [inline apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx:298]: We need to be careful with the usage of `useMemo` `useCallback` `useEffect`. it's actually not a good practice to overuse them and can worsen perfomance
It would be as efficient to not call `useCallback` here

```ts
const onCloseModal = () => setOpen(false);
```

https://kentcdodds.com/blog/usememo-and-usecallback

File: apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx
Comment (lambertkevin) [inline apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx:20]: Just a reminder to update this with the right link, cause I guess `ratingsContact` isn't going to help much 😝

File: apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx
Comment (lambertkevin) [inline apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx:309]: not important but since `isOpen` is a `boolean` that shouldn't be necessary, right ?

**Review by lambertkevin — APPROVED** [review]
Just 2 comments, but otherwise LGTM 🚀

File: apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx
Comment (lvndry) [inline apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx:20]: waiting on this to merge right ?

**Review by lvndry — DISMISSED** [review]

File: apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx
Comment (henrily-ledger) [reply apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx:20]: Yes, I don't have the link

File: apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx
Comment (henrily-ledger) [reply apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx:298]: Ok nice, i'll fix it, I was just copy pasting other code.

Comment (lambertkevin) [reply apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx]: Not true, please keep using useMemo & useCallbacks everywhere.

Comment (lambertkevin) [reply apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx]: Also it has already been discussed, we're following this ideology: https://attardi.org/why-we-memo-all-the-things/

Comment (haammar-ledger) [reply apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx]: @lvndry confirmed, using these hooks is encouraged in our codebase, nothing blocking here. Let's keep this debate outside of this PR. :wink:

File: apps/ledger-live-mobile/src/locales/en/common.json
Comment (haammar-ledger) [inline apps/ledger-live-mobile/src/locales/en/common.json:5371]: Is this the final, approved wording? I'm just noticing it doesn't match what's currently in the task's description, but maybe the final wording is still WIP waiting for approval?

Comment (henrily-ledger) [reply apps/ledger-live-mobile/src/locales/en/common.json:5371]: Screenshot 2022-11-04 at 16 29 08

I guess ticket is not up to date

**Comment by haammar-ledger** [comment]
Ready for QA.

**Pull request merged**
